### DISCUSSION TITLE
Multiselect column with aliases

### DISCUSSION THREAD

**Discussion by vasnt** [AppFlowy-IO/AppFlowy#3676] [Ideas]
In many cases, multi-select column may has choices more than 5. 
Column width becomes constraint if more than 2 choices are filled. 
So it is better if aliases/short abbreviations (user defined) are used for each choice. 

e.g.  while choosing/adding new choice, it shows full view. as below.
Abbreviation - its meaning

TD - Today
YD- Yesterday
ND- Next Day
NM- Next Month
NY- Next Year

but in grid view, it should show only below abbreviations
TD
YD
ND
NM 
NY

So, is it possible to add abbreviation & full form both while filling choices? (so it can act as in-place short guide for users)

**Comment by annieappflowy** [comment]
Hi @vasnt , thanks for your feature request. Bear with me if I understand it incorrectly.
Would widening the column work for you? You can hover on the header's left/right border to adjust the width:
<img width="218" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/12026239/570efa4e-26c8-45df-813e-2fcdbe842f73">

**Comment by vasnt** [comment]
1) Usually, there are more than 5 columns in table so it is not suitable to fit within screen size. So widening column is not an effective solution in the use case i mentioned. 
2) when multiple choices are done,  text is wrapped  if column is not widened, and for single row heighten table, wrapping text (at one place/column) dont look good 

e.g. such abbreviations can fit  a lot more. 
<img width="165" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/117589240/d3e833fb-37b0-49c6-8d54-472ce937d969">


or emojis can be used to be more concise (& visual too).
